Week-2/1.Flight-booking.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+function getFullName(firstname, surname, useFormalName, gender) {
+  firstname = firstname.trim();
+  surname = surname.trim();
+  const fullName = `${firstname} ${surname}`;
+
+  if (!firstname || !surname) {
+    return "Invalid Input";
+  }
+
+  if (!useFormalName) {
+    return fullName;
+  }
+
+  if (gender == "male") {
```

Comment:
You can do something like: if formal name --> gender === 'male' ? 'lord' : 'lady' --> append that to full name

this is a big improvement from the old code though :) Just showing what you missed

Reply:
yeah this what I was saying the one I forgot hahaha kasi felt super binary kasi haha eh well ganun tlga